test/data/ReqMgr/requests/Integration/SC_Nano.json

```diff
@@ -26,10 +26,10 @@
         "CMSSWVersion": "CMSSW_10_6_26",
         "Campaign": "Campaign-OVERRIDE-ME",
         "Comments": {
-            "CheckList": "StepChain with Dataset start policy",
+            "CheckList": "StepChain with Dataset start policy; EventAwareLumiBased splitting hardcoded to 10k EpJ/EpL",
             "WorkFlowDesc": [
                 "TC processing a MINIAODSIM, thus locking all of the input blocks at the same location (with copies=2);",
```

Comment:
The description should say "SC processing a MINIAODSIM" in place of "TC processing...", right?

Reply:
Good catch! Let me update this description as well.